front/src/applications/stdcm/hooks/useStaticPathfinding.ts

```diff
@@ -84,7 +84,7 @@ const useStaticPathfinding = (infra?: InfraWithState) => {
     [pathfinding, pathProperties]
   );
 
-  return result;
+  return { pathfinding: result, isFetching };
```

Comment:
Better to do that because when we use it on other component it's hard to follow what is actually fetching.
```suggestion
  return { pathfinding: result, isPathfindingLoading: isFetching };
```

Reply:
done